exla/lib/exla/computation.ex

```diff
@@ -70,6 +72,21 @@ defmodule EXLA.Computation do
     }
   end
 
+  def compile(
+        %EXLA.MLIR.Function{module: module, xla_return_shape: ret_shape},
```

Comment:
Can we just call it `return_shape`?

Reply:
now that we're removing MLIR-specific things like MLIR.Type, yes

Reply:
I removed MLIR.Type, and this is now just return_shape :)